Gems/Atom/RHI/Code/Include/Atom/RHI/SubmitItem.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+namespace AZ
+{
+    namespace RHI
+    {
+        /// Base for the command list Submit item types.
```

Comment:
Minor: I understand where for existing classes you are following the existing commenting convention but for newer classes lets follow the correct commenting standard which is //! for class/api definitions and // for everything else